client/signup/steps/survey/survey-step-v2.jsx

```diff
@@ -107,7 +107,7 @@ export default React.createClass( {
 
 	handleOtherWriteIn( e ) {
 		this.setState( {
-			otherWriteIn: e.target.value
+			otherWriteIn: e.target.value.replace( /^(\s|\.|\,|\:|\!|\?)+|(\s|\.|\,|\:|\!|\?)+$/g, '' ),
```

Comment:
>I'm down, but isn't [^\w]|\s redundant? (Shouldn't [^\w] include \s too?)

Sorry, I totally misunderstood the question. Yes, the `|\s` is redundant. Great catch.